### ISSUE TITLE
Still have the problem after download the taxonomy first

### ISSUE THREAD

**Issue by Tsingsjeen** [DerrickWood/kraken2#517]
@msnyderman Did you run `kraken2-build --download-taxonomy --db DB` before building the standard database?

@pe-se It seems like your system does not have more than 1 thread and so you cannot use the `--threads 32` option.

If you continue to have problems, please create a new issue.

_Originally posted by @jenniferlu717 in https://github.com/DerrickWood/kraken2/issues/54#issuecomment-611878820_

**Comment by frayzdecastello** [comment]
Nope. Did not work. Only plasmid and UniVec_Core were downloaded. Got error on archaea, bacteria, fungi, human, plant, protozoa and viral.

Tried all the recommendations by lexming or #508.

rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/839/185/GCF_000839185.1_ViralProj14174

rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/762/265/GCF_000762265.1_ASM76226v1

rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/002/945/GCF_000002945.1_ASM294v2

mv: replace 'assembly_summary.txt', overriding mode 0444 (r--r--r--)?
rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/405/GCF_000001405.39_GRCh38.p13

rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/317/415/GCF_000317415.1_Csi_valencia_1.0

rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/900/128/725/GCF_900128725.1_BCifornacula_v1.0

rsync_from_ncbi.pl: unexpected FTP path (new server?) for https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/194/455/GCF_000194455.1_ASM19445v1

**Comment by mnemosymenicolas** [comment]
Hello !

I'm using kraken2 v2.1.2 on a linux computer, I downloaded taxonomy first without any issue by when I'm using this command :
kraken2-build --standard --use-ftp --db database_kraken2

No matter if I change the "rsync_from_ncbi.pl" file from "^ftp://" to "^https://", the same error is occuring :
"rsync_from_ncbi.pl: unexpected FTP path (new server?)"

Even if I try to download : bacteria, viral library the issue still here.

If you have any solutions I would be grateful to use it. Thanks !
